sdk/objectanchors/Azure.MixedReality.ObjectAnchors/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 0.1.0-beta.0 (2021-03-01)
```

Comment:
@craigktreasure 
Changed to 0.1,0-beta.0
RE: "This doesn't need to match the API version. They also don't want us to use the -preview suffix in these SDKs anymore, so it should be -beta. Maybe 0.1.0-beta.0?"